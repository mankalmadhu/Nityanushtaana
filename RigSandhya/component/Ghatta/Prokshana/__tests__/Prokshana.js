import React from 'react';
import ಪ್ರೋಕ್ಷಣ from '../Prokshana';
import Renderer from 'react-test-renderer';

jest.mock('../../Ghatta.vinyasa');
jest.mock('../../../Vishaya/Vishaya',()=>'ವಿಷಯ');
jest.mock('../../../Vishaya/Sheershike',()=>'ಶೀರ್ಷಿಕೆ');

test("ಪ್ರೋಕ್ಷಣ", () => {
  expect(Renderer.create(
    <ಪ್ರೋಕ್ಷಣ />
  ).toJSON()).toMatchSnapshot();

});