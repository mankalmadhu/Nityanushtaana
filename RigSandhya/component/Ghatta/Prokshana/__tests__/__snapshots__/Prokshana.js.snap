// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ಪ್ರೋಕ್ಷಣ 1`] = `
<View>
  <ಶೀರ್ಷಿಕೆ>
    ಪ್ರೋಕ್ಷಣ
  </ಶೀರ್ಷಿಕೆ>
  <ವಿಷಯ
    ವಿನ್ಯಾಸ="ವಿನಿಯೋಗ"
  >
    ಅಪವಿತ್ರಃ ಪವಿತ್ರೋವಾ ಸರ್ವಾವಸ್ಥಾ೦ ಗತೋಽಪಿವಾ । ಯಸ್ಸ್ಮರೇತ್ ಪು೦ಡರೀಕಾಕ್ಷ೦ ಸಬಾಹ್ಯಾಭ್ಯ೦ತರಶ್ಶುಚಿಃ ।
  </ವಿಷಯ>
</View>
`;
